File: Exercism Codes/Java/SpaceAge.java
class SpaceAge {

    private double seconds;
    
    SpaceAge(double seconds) {
        this.seconds = seconds / 31557600;
    }

    double getSeconds() {
        return seconds;
    }

    double onEarth() {
        return seconds / 1.0;
    }

    double onMercury() {
        return seconds / 0.2408467;
    }

    double onVenus() {
        return seconds / 0.61519726;
    }

    double onMars() {
        return seconds / 1.8808158;
    }

    double onJupiter() {
        return seconds / 11.862615;
    }

    double onSaturn() {
        return seconds / 29.447498;
    }

    double onUranus() {
        return seconds / 84.016846;
    }

    double onNeptune() {
        return seconds / 164.79132;
    }

}

File: Exercism Codes/C#/SpaceAge.cs
using System;

public class SpaceAge
{
    private int seconds;

    public SpaceAge(int seconds)
    {
        this.seconds = seconds;
    }

    public double OnEarth()
    {
        return seconds / (1.0 * 31557600);
    }

    public double OnMercury()
    {
        return seconds / (0.2408467 * 31557600);
    }

    public double OnVenus()
    {
        return seconds / (0.61519726 * 31557600);
    }

    public double OnMars()
    {
        return seconds / (1.8808158 * 31557600);
    }

    public double OnJupiter()
    {
        return seconds / (11.862615 * 31557600);
    }

    public double OnSaturn()
    {
        return seconds / (29.447498 * 31557600);
    }

    public double OnUranus()
    {
        return seconds / (84.016846 * 31557600);
    }

    public double OnNeptune()
    {
        return seconds / (164.79132 * 31557600);
    }
}